java/daikon/FileIO.java

```diff
@@ -1805,6 +1811,10 @@ public static void process_sample(
       return;
     }
 
+    // Workaround for Checker Framework issue #862
+    // https://github.com/typetools/checker-framework/issues/862
+    // Use NIS so that it is in scope to check the precondition below.
+    Object o = NIS.suppressor_map;
```

Comment:
If this isn't used, I would name it something like dummy rather than o which is often used for real variables.